handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/filter/StageDistinguishFilter.java

```diff
@@ -0,0 +1,30 @@
+package io.servicecomb.loadbalance.filter;
+
+import com.netflix.loadbalancer.Server;
+import io.servicecomb.loadbalance.CseServer;
+import io.servicecomb.loadbalance.ServerListFilterExt;
+import io.servicecomb.serviceregistry.RegistryUtils;
+import io.servicecomb.serviceregistry.api.registry.MicroserviceInstance;
+import io.servicecomb.serviceregistry.api.registry.MicroserviceInstanceStatus;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StageDistinguishFilter implements ServerListFilterExt {
```

Comment:
please use DiscoverFilter to impl this

and i heard that, SC will only return same stage instances, so this filter is useless?

Reply:
this feature given by SC, they will return all instances of different env's. STAGE is another scope of separation.